### PR TITLE
RHOAIENG-72: Specify numeric UID in Dockerfiles

### PR DESCRIPTION
**Pull request by jiridanek**
https://issues.redhat.com/browse/RHOAIENG-72

This resolves the

```
Error: container has runAsNonRoot and image has non-numeric user (rhods), cannot verify user is non-root
```

issue.

OpenShift (by default)
runs images under a random user id and the root user group.

See https://access.redhat.com/documentation/cn/openshift_container_platform/4.15/html/images/creating-images#use-uid_create-images

## Merge criteria:

- [x] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: components/notebook-controller/Dockerfile
Comment (jiridanek) [reply components/notebook-controller/Dockerfile:60]: OpenShift default is to set random uid and 0 gid, https://access.redhat.com/documentation/cn/openshift_container_platform/4.15/html/images/creating-images#use-uid_create-images

I feel like we should keep close to that

Comment (jstourac) [reply components/notebook-controller/Dockerfile:60]: Well, I read this link before, since you have it in the description... so I probably missed something here.
My point is - why setting also GID here as it is 0 by default anyway. From your answer, I understand that it's just a precaution, correct?

**Comment by jstourac** [comment]
FTR https://issues.redhat.com/browse/RHOAIENG-72

File: components/notebook-controller/Dockerfile
Comment (jiridanek) [reply components/notebook-controller/Dockerfile:60]: Exactly.

**Review by jstourac — APPROVED** [review]

**Comment by atheo89** [comment]
thanks for your patience waiting for approval!
/approve

**Comment by jiridanek** [comment]
Thanks!

**Pull request merged**
